apps/molgenis-components/src/client/client.ts

```diff
@@ -42,7 +42,7 @@ export default {
         return deepClone(schemaMetaData).tables.find(
           (table: ITableMetaData) =>
             table.id === convertToPascalCase(tableName) &&
-            table.externalSchema === schemaName
+            table.externalSchema === schemaNameCache
```

Comment:
If you don't provide a schemaName, it will ask the graphql endpoint and retrieve this name. E.g. if you run the client in the 'catelogue' schema it will be 'catalogue' and if you run in 'pet store' it will be 'pet store. This disconnects your code from schemaName which we need in this case as this app can run on all copies of data catalogue.

Max came up with the name 'schemaNameCache' <= I simply updated the schemaName but that is kinda nasty as that is parameter value and not a variable within the object.